lib/commands/gesture.js

```diff
@@ -130,6 +130,10 @@ commands.performActions = async function performActions (actions) {
   return await this.proxyCommand('/actions', 'POST', {actions: preprocessedActions});
 };
 
+commands.releaseActions = function releaseActions () {
```

Comment:
should it be async

Reply:
also it probably won't work if the route is not present in base driver

Reply:
the route is present in our routes.js. it's just not implemented in this driver.

Reply:
> should it be async?
theoretically  yes. but since it's a no-op function, making it async would run afoul of our linting which would require an await. if you `await` a non async function it just gets you the return value, so there's no harm in it not being async.

Reply:
> the route is present in our routes.js. it's just not implemented in this driver.

Could you please point me to the place where it is defined in the base driver? I'm unable to find it there

Reply:
https://github.com/appium/appium/blob/2.0/packages/base-driver/lib/protocol/routes.js#L281-L283

Reply:
oh, we had.
https://github.com/appium/appium-base-driver/blob/master/lib/protocol/routes.js will support this only in 2.0 or will we add it in 1.x for our next 1.x (1.22.0) release?